exporter/datadogexporter/internal/hostmetadata/metadata.go

```diff
@@ -20,10 +20,11 @@ import (
 	ec2Attributes "github.com/DataDog/opentelemetry-mapping-go/pkg/otlp/attributes/ec2"
 	"github.com/DataDog/opentelemetry-mapping-go/pkg/otlp/attributes/gcp"
 	"github.com/DataDog/opentelemetry-mapping-go/pkg/otlp/attributes/source"
+	"go.uber.org/zap"
```

Comment:
Format is off

Reply:
oddly my IDE really prefers this.  no clue why.  fixed